packages/@divvi/mobile/src/components/RowDivider.tsx

```diff
@@ -1,20 +1,30 @@
 import * as React from 'react'
 import { StyleSheet, View } from 'react-native'
-import Colors, { ColorValue } from 'src/styles/colors'
+import themeColors, { type ColorValue } from 'src/styles/colors'
 import { Spacing } from 'src/styles/styles'
 
 export interface Props {
   color?: ColorValue
+  spacing?: Spacing
+  testID?: string
 }
 
-export default function RowDivider({ color = Colors.borderPrimary }: Props) {
-  return <View style={[styles.container, { backgroundColor: color }]} />
+export default function RowDivider({
+  color = themeColors.borderPrimary,
+  spacing = Spacing.Regular16,
+  testID,
+}: Props) {
+  return (
+    <View
+      testID={testID}
+      style={[styles.container, { backgroundColor: color, marginVertical: spacing }]}
+    />
+  )
```

Comment:
Renaming props for clarity also requires changing the use in `./components/FeeInfoBottomSheet.tsx`.

```suggestion
export default function RowDivider({
  backgroundColor = themeColors.borderPrimary,
  marginVertical = Spacing.Regular16,
  testID,
}: Props) {
  return (
    <View
      testID={testID}
      style={[styles.container, { backgroundColor, marginVertical }]}
    />
  )
```